packages/ui/src/components/vuestic-components/va-button/VaButton.vue

```diff
@@ -223,7 +223,7 @@ export default class VaButton extends mixins(
       computedStyle.boxShadow = this.shadowStyle
     }
 
-    if (this.context.va.color && !this.outline && !this.flat) {
+    if (this.context?.va?.color && !this.outline && !this.flat) {
```

Comment:
can you use `useColor`? Or whatever we need to interact with color more directly.
va.color more intended for users.